app/src/main/java/ca/bc/gov/bchealth/utils/StringExtensions.kt

```diff
@@ -7,5 +7,8 @@ import androidx.core.text.HtmlCompat
 fun String?.orPlaceholder(placeholder: String = "--"): String =
     this ?: placeholder
 
+fun String?.orPlaceholderIfNullOrBlank(placeholder: String = "--"): String =
```

Comment:
This is duplicate of the above method declared String?.orPlaceholder
and also has un-nacessary if condition check.

Reply:
They are actually different: `orPlaceholder` will use placeholder only if the content is null, `orPlaceholderIfNullOrBlank` will also return the placeholder if the content is blank